app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    @user = current_user
+  end
+
+  def create
+    @post = Post.find(params[:post_id] || params[:id])
+    @comment = Comment.new(text: params[:comment][:text], post_id: @post.id, author_id: current_user.id)
```

Comment:
- [ ] Just like in the posts controller above, let's make use of the strong parameters as permitted by the comment_params below:

  ```
  @comment = Comment.new(comment_params)
  ```